### ISSUE TITLE
Hide that the website is using the Disciple.Tools theme.

### ISSUE THREAD

**Issue by corsacca** [DiscipleTools/disciple-tools-theme#615]
**Purpose**
Make it hard to tell that a instance is running the Disciple.Tools theme. 

**Use case** 
This for teams or orgs operating in harder contexts who want to keep the nature of their activity hidden. If someone get a hold of the link to the team's D.T instance, they be able to tell what the website is used for.

**Requirements**
- change the folder structure so assets aren't loaded from `https://example.com/wp-content/themes/disciple-tools-theme/...`
- on the login page and any other publicly available page: change css and ids that include "disciple" or other similar words

**Code considerations**
This functionality could be added to the Theme customization plugin: https://github.com/DiscipleTools/disciple-tools-theme/issues/730. So of the core theme code might need to be configured as well.

**Security considerations**
It might be impossible to fully hide that the site is using the disciple-tools-theme. The code is all open source and so is publicly searchable on the web.

**Comment by jlebouch76** [comment]
Yes this is something we will definitely need to have on both webpage and mobile app after an user logged in. 
Thanks

**Comment by corsacca** [comment]
I update the description of this issue to be the security aspect of white-labeling.
For the cosmetic side of customizing the theme see: https://github.com/DiscipleTools/disciple-tools-theme/issues/730

**Comment by corsacca** [comment]
This has been done as much as possible with D.T and Wordpress
